### PR TITLE
made 8ball cmd

### PR DESCRIPTION
**Pull request by RopraMMC**
Made a 8ball command - followed 8-PEP guidelines, added a description to Russian roulette

### CODE REVIEW

**Comment by Dekriel** [comment]
I mean you can do the randomisation of the list in the embed but ok

File: src/cogs/fun.py [hunk 1]
```diff
@@ -190,15 +190,29 @@ async def casino(self, ctx: Context) -> None:
...
+                                       "definitely hates life", "plays this game 24/7", "has issues",
                                        "probably needs some help"])
-        embed = Embed(description=random_choice,colour=embed_colour[random_choice]).set_footer(
-            text=f"{ctx.author.name} {random_footer}", icon_url=(ctx.author.display_avatar.url))
+        embed = Embed(description=random_choice, colour=embed_colour[random_choice]).set_footer(
+            text=f"{ctx.author.name} {random_footer}", icon_url=ctx.author.display_avatar.url)
         return await ctx.send(embed=embed)
 
+    @command(aliases=["8ball", "8_ball", "eightball"])
+    async def eight_ball(self, ctx: Context, *, message):
```
Comment (Pandabweer) [inline src/cogs/fun.py:204]: typehint message

**Review by Pandabweer — APPROVED** [review]

File: src/cogs/fun.py (hunk 1 above)
Comment (RopraMMC) [reply src/cogs/fun.py:204]: _psst  what does that mean?_

**Review by Pandabweer — CHANGES_REQUESTED** [review]
wups

**Review by Dekriel — APPROVED** [review]
it's ok

File: src/cogs/fun.py (hunk 1 above)
Comment (RopraMMC) [reply src/cogs/fun.py:204]: _is it the ##?_

**Pull request closed without merging**